src/shared/directives/conditional-show/conditional-show.directive.ts

```diff
@@ -17,7 +17,8 @@ import { ConditionalShowRegistrarService } from './services/conditional-show-reg
 export class ConditionalShowDirective implements AfterViewInit, OnDestroy {
 
   @Input() caseField: CaseField;
-  @Input() eventFields: CaseField[] = [];
+  @Input() idPrefix: string;
+  @Input() caseFields: CaseField[] = [];
```

Comment:
👍  for renaming, but the new name is not entirely correct. `conditional-show.directive.ts` can be applied in different contexts. For example to tabs, and in that case the `caseFields` array will only contain the fields declared in the tab. I think something like `contextFields` might be more appropriate.